lib/Server.js

```diff
@@ -580,17 +581,91 @@ class Server {
   }
 
   showStatus() {
-    const suffix = '/';
-    const uri = `${createDomain(this.options, this.server)}${suffix}`;
+    const useColor = getColorsOption(getCompilerConfigArray(this.compiler));
 
-    const configArr = getCompilerConfigArray(this.compiler);
-    const colors = getColorsOption(configArr);
+    const protocol = this.options.https ? 'https' : 'http';
+    const { hostname, port } = this;
+    const prettyPrintUrl = (newHostname) =>
+      url.format({ protocol, hostname: newHostname, port, pathname: '/' });
+
+    let prettyHostname;
+    let lanUrlForTerminal;
+
+    if (hostname === '0.0.0.0' || hostname === '::') {
+      prettyHostname = 'localhost';
+
+      const localIP =
+        hostname === '::' ? internalIp.v6.sync() : internalIp.v4.sync();
+
+      if (localIP) {
+        lanUrlForTerminal = prettyPrintUrl(localIP);
+      }
+    } else {
+      prettyHostname = hostname;
+    }
 
-    status(uri, this.options, this.logger, colors);
+    const localUrlForTerminal = prettyPrintUrl(prettyHostname);
+
+    if (lanUrlForTerminal) {
+      this.logger.info('Project is running at:');
+      this.logger.info(`Local: ${colors.info(useColor, localUrlForTerminal)}`);
+      this.logger.info(
+        `On Your Network: ${colors.info(useColor, lanUrlForTerminal)}`
+      );
+    } else {
+      this.logger.info(
+        `Project is running at ${colors.info(useColor, localUrlForTerminal)}`
+      );
+    }
+
+    if (
+      this.options.dev &&
+      typeof this.options.dev.publicPath !== 'undefined'
+    ) {
+      this.options.info(
```

Comment:
@alexander-akait looks like `this.options.info` should be replaced with `this.logger.info`

Reply:
Already fixed in master